guides/release/upgrading/index.md

```diff
@@ -1 +1,81 @@
+An edition is essentially a collection of features that are available in a minor (non-breaking) release of Ember.
+Although it's possible to do a big-bang refactor to change an app to use all of Octane's features at once, it's really designed for incremental adoption.
+You can make changes at the pace that makes sense for your project or team.
+
+Read on to learn how to begin using Octane's features in an existing application.
+In summary:
+
+1. Follow the [regular upgrade steps](https://cli.emberjs.com/release/basic-use/upgrading/) to update your app to at least version `3.x`. <!-- TODO -->
+2. Run your tests to make sure everything still works as expected
+3. Review the deprecation warnings and make any necessary refactors. Some deprecations will need to be resolved before you can use an Octane feature. Read on to learn what they are.
+4. Go through the [Quick start tutorial](../../getting-started/quick-start) to learn the fundamentals of Octane. You might need to study up on [Native JavaScript Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/class) too.
+5. Install `@ember/optional-features` in your app, if it is not already in the `devDependencies` of your `package.json`
+6. Turn on optional features one by one, running tests in between to make sure things still work as expected. Read on to learn what an Octane configuration looks like.
+7. Create a new component in your app, and experiment!
+8. Try refactoring some existing code to use a new feature. Check out the [cheat sheet](../cheat-sheet/) and [Editions deep dive](../editions/) for some pointers.
+8. Schedule a show-and-tell meeting with your coworkers to fill them in about what you learned 
+9. Review the refactoring checklist below to create a plan for handling existing code
+
+If you need help along the way, visit [the Ember Community chat and forums](https://emberjs.com/community/).
+
+### Deprecations that matter for Octane
+
+- [`jquery-apis`](https://deprecations.emberjs.com/v3.x/#toc_jquery-apis) - By default, Octane does not include jQuery. Continuing to use jQuery in your app will not conflict with Octane features, however you should follow the deprecation instructions to keep using it.
+
+<!-- TODO -->
+
+### Optional features in Octane
+
+A fully-Octane app has the following configuration in `config/optional-features.json`:
+
+```json
+{
+  "application-template-wrapper": false,
+  "jquery-integration": false,
+  "template-only-glimmer-components": true
+}
+```
+
+Use the command `ember feature:list` in your console to learn what each option does.
+
+### Refactoring checklist
+
+For many of the optional features, the thing they affect the most is what you see in newly created files, not your existing code.
+That's a good place to start, but eventually you may want to refactor existing code so that your app follows one main programming model, not a mixture of Octane and Classic.
```

Comment:
Should we mention something here about how, even though your app will keep working on Octane even though it isn't completely idiomatic, you probably want to follow Octane idioms (and hence it's good to refactor in that direction to use those new idioms)?